config/locales/member_facing.fr.yml

```diff
@@ -119,3 +119,15 @@ fr:
   time:
     formats:
       default: "%-d %B %Y"
+  confirmation_mailer:
+    confirmation_email:
+      subject: "E-mail confirmation for signing up for express donations with SumOfUs"
+      headline: "Thank you for your donation!"
+      visit_this: "To confirm your enrollment in our express donations plan, visit this"
+      link: "link"
+      url: ", or copy paste this URL to your browser: %{confirmation_url}"
```

Comment:
Couldn't this just be a single entry with interpolation - you'd pass in the anchor tag like you have with the confirmation url